### PR TITLE
homework 9 Python classes

### PR DESCRIPTION
**Pull request by OlenaNechai**

### CODE REVIEW

File: homework_9/company.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    def key_people(self):
+        return self.__key_people
+
+    @property
+    def total_employees(self):
+        return self.__total_employees
+
+    @total_employees.setter
+    def total_employees(self, new_total: int):
+        self.__total_employees = new_total
```
Comment (bohdandovbsyh) [inline homework_9/company.py]: Looks like I can set a negative number for this property, or event smth that is not integer

File: homework_9/worker.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    def name(self):
+        return self.__name
+
+    @property
+    def surname(self):
+        return self.__surname
+
+    @property
+    def position(self):
+        return self.__position
```
Comment (bohdandovbsyh) [inline homework_9/worker.py:28]: Actually position of worker can be changed/ So it is good to have setter for it

File: homework_9/worker.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    def annual_bonus(self):
+        if self.__position == 'Employee':
+            return 1000
+        elif self.__position == 'Lead':
+            return 2000
+        else:
+            return 3000
+
+    def salary_info_for_worker(self):
```
Comment (bohdandovbsyh) [inline homework_9/worker.py]: General comment about function naming. It is good practice to start the name of the function with  verb.
For example:
get_salary_info....
calculate_anual_bonus..

File: homework_9/worker.py (hunk 2 above)
Comment (OlenaNechai) [reply homework_9/worker.py:28]: I've added the setter on line 30

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**